server-kotlin/src/city_sync/model/CityBasic.kt

```diff
@@ -0,0 +1,10 @@
+package city_sync.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
(detail) JsonIgnoreProperties can be configured globally in Jackson mapper config